en/micro-integrator/docs/references/connectors/servicenow-connector/settingup-servicenow-instance.md

```diff
@@ -0,0 +1,24 @@
+# Setting up the ServiceNow Instance
+
+The ServiceNow connector allows you to access the ServiceNow REST API of through WSO2 EI. To use the REST API, we need to have a [Servicenow](https://www.servicenow.com/) account. 
+
+## Signing Up for Servicenow
+
+1. Visit [Servicenow](https://www.servicenow.com/) site and sign up for an account and do the verification step. 
+  <img src="/assets/img/connectors/servicenow-1.png" title="Creating Servicenow ID" width="800" alt="Creating Servicenow ID"/>
+
+2. Now you need to create an instance. To do that visit [Servicenow Developer account](https://developer.servicenow.com/dev.do) with the credentials you obtain at step1. Click on **Request Instance** button. 
+  <img src="/assets/img/connectors/servicenow-2.png" title="Login to Developer Account" width="800" alt="Login to Developer Account"/>
+
+3. Once you reach to **Request an Instance** page, you can choose your release. 
```

Comment:
```suggestion
3. Once you reach the **Request an Instance** page, you can choose your release. 
```
What do we mean by "release" here? Can we add more clarification?

Reply:
Added